### PR TITLE
Introduce support for authzed

### PR DESCRIPTION
**Pull request by iocanel**
The pull request implements a RelationshipManager based on authzed with all needed bells and whistles.

### CODE REVIEW

**Comment by kdubb** [comment]
Awesome... gonna look this over and merge tomorrow

File: zanzibar-authzed/deployment/src/test/java/io/quarkiverse/zanzibar/openfga/test/ZanzibarOpenFGADevModeTest.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusDevModeTest;
+
+public class ZanzibarOpenFGADevModeTest {
```
Comment (kdubb) [inline zanzibar-authzed/deployment/src/test/java/io/quarkiverse/zanzibar/openfga/test/ZanzibarOpenFGADevModeTest.java:11]: Class & file should be named for Authzed

File: zanzibar-authzed/deployment/src/test/java/io/quarkiverse/zanzibar/openfga/test/ZanzibarOpenFGATest.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusUnitTest;
+
+public class ZanzibarOpenFGATest {
```
Comment (kdubb) [inline zanzibar-authzed/deployment/src/test/java/io/quarkiverse/zanzibar/openfga/test/ZanzibarOpenFGATest.java:11]: Class & file should be named for Authzed

File: zanzibar-authzed/runtime/src/main/resources/META-INF/quarkus-extension.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+name: Zanzibar Openfga
```
Comment (kdubb) [inline zanzibar-authzed/runtime/src/main/resources/META-INF/quarkus-extension.yaml:1]: Need to reference Authzed here

**Review by kdubb — CHANGES_REQUESTED** [review]
Need to cleanup some copy/paste remnants from OpenFGA adapter.

**Comment by kdubb** [comment]
The README should be updated as well to include support for Authzed.

**Comment by kdubb** [comment]
@iocanel I'm looking to get Zanzibar "ready" for Quarkus 3. Can we get this cleaned up so I can merge it? I'm working on docs and a quickstart it would be nice to support both OpenFGA and Authzed for a `1.0` release.

**Comment by iocanel** [comment]
Ok

**Comment by iocanel** [comment]
@kdubb: applied feedback.

**Pull request merged**